minishift/Makefile

```diff
@@ -1,43 +1,59 @@
 # Setup WIT image URL, use default image path and default tag if not provided
-WIT_IMAGE_DEFAULT=docker.io/fabric8/fabric8-wit
```

Comment:
Without `docker.io/` we can use local images as well. Docker will search for images in local registry and then docker hub

Reply:
hmm but that will need the team to create image locally, right?

Reply:
No. Docker searches a number of repositories, include `local`, `hub.docker`, `registry.centos`, etc

Reply:
ok got it